engine/src/main/scala/cromwell/webservice/metadata/MetadataBuilderActor.scala

```diff
@@ -131,7 +133,9 @@ object MetadataBuilderActor {
 
   /** Sort events by timestamp, transform them into TimestampedJsValues, and merge them together. */
   private def eventsToIndexedJson(events: Seq[MetadataEvent]): TimestampedJsValue = {
-     events map { e => keyValueToIndexedJson(e.key.key, e.value, e.offsetDateTime) } reduce(_ |+| _)
+    // The `List` has a `Foldable` instance defined in scope, and because the `List`'s elements have a `Monoid` instance
+    // defined in scope, `suml` can derive a sane `TimestampedJsValue` value even if the `List` of events is empty.
+    events.toList map { e => keyValueToIndexedJson(e.key.key, e.value, e.offsetDateTime) } suml
```

Comment:
TOL and not a hold back for this PR at all. I think `suml` is a foldLeft behind the hood. The advantage of `reduce` (even if we don't really use it here - but we could with a parallel collection or something else) is that the "reducing" can be done in parallel because the `TimestampedJsValue` Monoid is now commutative. Whereas the `foldLeft` will be done sequentially.